src/interface/api/Output.cpp

```diff
@@ -113,9 +104,17 @@ void Output::setInterfaceGameState(GameState interfaceGameState) {
     Output::interfaceGameState = interfaceGameState;
 }
 
-const pidVals &Output::getShotControllerPID() { return shotControllerPID; }
+const pidVals &Output::getNumTreePid() { return numTreePID; }
+
+void Output::setNumTreePid(const pidVals &numTreePid) { numTreePID = numTreePid; }
+
+const pidVals &Output::getReceivePid() { return receivePID; }
+
+void Output::setReceivePid(const pidVals &receivePid) { receivePID = receivePid; }
```

Comment:
All these methods should be documented and explain what the meaning is of the value returned or set.